addons/html_editor/static/src/main/table/table_plugin.js

```diff
@@ -534,8 +535,14 @@ export class TablePlugin extends Plugin {
     }
 
     updateSelectionTable(selectionData) {
-        if (this.hanldeFirefoxSelection()) {
+        if (
+            this.hanldeFirefoxSelection() ||
+            this._isFirefoxDoubleMousedown ||
+            this._tripleClickSelection
+        ) {
             // It will be retriggered with selectionchange
+            delete this._isFirefoxDoubleMousedown;
+            delete this._tripleClickSelection;
```

Comment:
Actually, I would add `else` clause in `onMousedown` below to set those flags to `false` when their respective condition is not met and I would not delete the keys here. By doing that we have the information if we need it elsewhere, notably `this._isTripleClickSelection` which will be useful in the future.

Reply:
That will work but we should consider this case that after selecting cell by tripleclick if user changes selection without using mouse (By using arrow keys or something like that) and navigate to another cell then `updateSelectionTable` will be returned as `this._isTripleClickSelectioncell` is true and cell will not get deselected. If we really want to do this then we also have to set the value of the flag to `false` in `keydown` listener.

Reply:
@adsa-odoo Oh right, right. And then it would have to be in the selection plugin, and share a getter function and all that stuff... Ok you're right it's too complicated. Let's just rename it to `_isTripleClickInTable` so we can delete it in table just like you did.

Reply:
@dmo-odoo Done changes!

Reply:
@adsa-odoo You did not rename `_isTripleClickSelection` to `_isTripleClickInTable`.

Reply:
@dmo-odoo  Sorry about that! I though for the previous comment :) Now it is updated, thanks!

Reply:
@adsa-odoo Thank you!